packages/core-cordova/www/metrics.js

```diff
@@ -0,0 +1,57 @@
+const MetricsService = cordova.require("core.core").MetricsService;
```

Comment:
Is this going to stay as "core.core" ? We might need to revise this when trying to publish the plugin.

Reply:
Yeah. We pushed that to the list so it will change.

Reply:
The first core is the plugin, the second is the module. It could be "aerogear.core"